kore/src/Branch.hs

```diff
@@ -16,7 +16,7 @@ module Branch
     , alternate
     ) where
 
-import Prelude.Kore
+import Prelude
```

Comment:
Why did you replace the `Prelude.Kore` here and in ListT.hs ?

Reply:
These two modules were originally written for the default `Prelude`, and they don't rely on anything we use in `Prelude.Kore`. I was going to have to write a lot of `hiding (...)` statements, so I decided to go back to the default `Prelude` instead.